test/runnable/ice10086b.d

```diff
@@ -5,3 +5,53 @@ import imports.ice10086y;
 import imports.ice10086x;
 
 void main() { test(); }
+
+static if (0)
```

Comment:
Remove condition?

Reply:
No, I just wanted some code so someone looking into how OPframeptr could be fixed with inlining would have a reduced test case handy.